Source/WebKit/Shared/LayerTreeContext.cpp

```diff
@@ -32,23 +32,13 @@
 
 namespace WebKit {
 
-LayerTreeContext::LayerTreeContext()
-    : contextID(0)
-{
-}
+LayerTreeContext::LayerTreeContext() = default;
 
-LayerTreeContext::~LayerTreeContext()
-{
-}
+LayerTreeContext::~LayerTreeContext() = default;
```

Comment:
Do we even need to spell this out? Why not just delete the code?

Reply:
I wasn’t sure what the intention was. If I remove the code then the constructor and destructor will be inline. It seems like whoever wrote this went out of their way to move them out of line so I kept that.

That said, I don’t really see the point of not having them inline so I’ll drop the implementation unless someone tells me otherwise.

Reply:
Typically we move them out of line when we can compile the header with fewer includes that way.

Reply:
Yes, though definitely not the case here.